### PR TITLE
Adding arc3o package

### PR DESCRIPTION
**Pull request by ClimateClara**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ClimateClara** [comment]
@conda-forge-admin, please ping team

File: recipes/arc3o/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ClimateClara/arc3o
+  summary: An observation operator for the Arctic Ocean for 6.9 GHz
+  dev_url: https://github.com/ClimateClara/arc3o
+  doc_url: https://arc3o.readthedocs.io/
+  license: GPL-3.0
```
Comment (dopplershift) [inline recipes/arc3o/meta.yaml:43]: Should be `GPL-3.0-only` or `GPL-3.0-or-later`

**Review by dopplershift — CHANGES_REQUESTED** [review]
This is in pretty good shape. Just need to tweak the license to make the linter happy.

File: recipes/arc3o/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (synapticarbors) [inline recipes/arc3o/LICENSE.txt:1]: You can remove this license file since one is packaged in the source distribution.

Comment (ClimateClara) [reply recipes/arc3o/LICENSE.txt]: thanks! done!

File: recipes/arc3o/meta.yaml (hunk 1 above)
Comment (ClimateClara) [reply recipes/arc3o/meta.yaml]: done! thanks!

**Comment by synapticarbors** [comment]
I've made the changes to the pathos feedstock that will fix the `pip check` failures here: https://github.com/conda-forge/pathos-feedstock/pull/14

**Pull request merged**